README.md

```diff
@@ -1,6 +1,45 @@
-# hypermodules
+<h1 align="center">✨<a href="https://github.com/hypermodules">hypermodules</a>⚡️</h1>
 
-- [github.com/hypermodules](https://github.com/hypermodules)
-- [github.com/hypermodules/hyperamp](https://github.com/hypermodules/hyperamp)
-- [github.com/hypermodules/dti](https://github.com/hypermodules/dti)
-- [github.com/hypermodules/entypo](https://github.com/hypermodules/entypo)
```

Comment:
Didn't have time to make screenshots, but I'm open to adding them back when someone has time to with screenshots.